uiplib/daemon.py

```diff
@@ -0,0 +1,138 @@
+#!/usr/bin/python
```

Comment:
I don't use `/usr/bin/python`. People like me, using `virtualenv`, can have issues with that.